TESTING

```diff
@@ -0,0 +1,404 @@
+// Include NeoPixel library
+#include <Adafruit_NeoPixel.h>
+
+// Define the PIN the strip is connected to, and the # of LEDs
+#define PIN      6
+#define N_LEDS 240
+
+// Define constant variables
+const int buttonPin = 2;
+const int buttonDelay = 150;
+
+// Define changing variables
+unsigned long previousMillis = 0;       // snapshot of time at the beginning of delay function
+unsigned long lastDebounceTime = 0;     // time of last button press
+int buttonState = 0;                    // whether button is HIGH or LOW
+int mode = 0;                           // the LED mode (currently 6 available)
```

Comment:
I recommend creating an enum for the mode variable, that way I can immediately know what each mode is.

https://stackoverflow.com/questions/4709175/what-are-enums-and-why-are-they-useful